docs/competition/teams/role-descriptions/reception.md

```diff
@@ -9,49 +9,63 @@ original:
 
-**Note: Training on how to use the scanning software will be given on the morning by the competition team coordinator.**
-
 ![Reception Desk Flow](../diagrams/reception-desk-flow.svg)
 
 At the start of the competition day, there will be four reception staffers
 covering the entrance. Once the initial influx of people is complete, this
-will reduce to two. Additionally, after the initial influx, receptionists
+will reduce. Additionally, after the initial influx, receptionists
 should allow people wearing SR wristbands to enter the building at will (they
 should check that people walking in have one, or follow the procedure above).
 
-**Note: All receptionists at the start of Saturday will be handling the arrivals desk too.**
-
-Throughout the competition reception staffers will encounter a variety of
+Throughout the competition, reception staffers will encounter a variety of
 members of the public enquiring what the event is, or otherwise receiving
-questions. Please answer as fully as possible, but refer any technical questions
-to helpdesk, or other unknown matters to other people in SR. Any member
-of the public should be allowed into the building if they sign a media consent
+questions. Please answer as fully as possible, but refer any technical or kit questions
+to the helpdesk. Any member of the public should be allowed into the building if they sign a media consent
 form.
 
-The 'crimper' role will also be performed by the receptionist. Please read this document as well for guidance on how to crimp correctly.
+# Handling Team Arrivals
+When a team first arrives or leaves, your role as reception is to log this and confiscate items on entry, see below:
+
+* Team entry/exit recording
+ 	* Arrival desk staff will have sign-in / sign-out sheet for team supervisors to
+record that their teams have arrived and when they leave. Reception will direct
+team supervisors and robot kit to the desk; you should get the team supervisor to sign the sheet.
+* Removing batteries, chargers, and bags:
+ 	* When on-site, we handle battery charging and robot safety, this means we should take all of the following items from each team:
+ 		* 2 x Battery
+ 		* 1 x Battery chargers
+		* 1 x Battery charging bags
+		* 1 x Battery charging power supply unit (PSU)
+
 
-After the initial influx of people, the 'arrival-desk' role
-will be absorbed by the receptionist. If you have a time slot after the first
-slot of the day, read that role description too.
+The following flow is an outline of the processes described above:
 
-## Required items
+![Arrival Desk Flow (Arrivals)](../diagrams/arrival-desk-arrivals-flow.svg)
+
+# Volunteer Arrivals
+Volunteers will be turning up throughout the competition, it's important to make sure they do what is needed before entering:
+- Strongly recommend that they wear a face covering.
+- Ensure they have filled out an MCF.
+- Ensure they have signed the safeguarding policy.
```

Comment:
"signed" isn't exactly the right word. Also, most people should have done this ahead of time, bear in mind we shouldn't really have any volunteers in the building at the same time as competitors who haven't signed the safeguarding policy, so this could become rather a blocker before we can let competitors in